include/net/net_pkt.h

```diff
@@ -2399,6 +2399,15 @@ static inline int net_pkt_write_le32_new(struct net_pkt *pkt, u32_t data)
 	return net_pkt_write_new(pkt, &data_le32, sizeof(u32_t));
```

Comment:
In commit called "net/pkt: Add a function to know the amount of data to read left", the subject is not understandable. Could we say here "net/pkt: Add a function to know the amount of remaining data" instead

Reply:
Sure. btw any idea on a better name for that function?